minikube/tb-redis-configmap.yml

```diff
@@ -0,0 +1,28 @@
+#
+# Copyright © 2016-2020 The Thingsboard Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: tb-redis-config
+  namespace: thingsboard
+  labels:
+    name: tb-redis-config
+data:
+  # Make sure that the value does not contain the port (:6379).
+  CACHE_TYPE: redis
+  REDIS_CONNECTION_TYPE: standalone
+  REDIS_HOST: tb-redis
```

Comment:
Please enable a non-default pool to unlock a performance. Default pool in only 8 connections wide.
```
  #REDIS POOL CONFIG
  REDIS_USE_DEFAULT_POOL_CONFIG: "false"
```